app/Resources/views/User/listCharacters.html.twig

```diff
@@ -1,43 +1,27 @@
 <div class="card">
     <div class="card-content">
-        <span class="card-title">
+        <div class="col s12 card-title">
             <h5>Verknüpfte Charaktere:</h5>
```

Comment:
You can use something like ```<h2 class="h5">...```
Also see http://v4-alpha.getbootstrap.com/content/typography/#headings